sdk/python/kfp/components/executor.py

```diff
@@ -238,8 +250,12 @@ def execute(self):
           func_kwargs[k] = self._get_output_parameter_path(k)
         else:
           func_kwargs[k] = self._get_output_artifact_path(k)
+      elif isinstance(v, OutputUri):
```

Comment:
We're assuming here that InputUri and OutputUri can only be used for artifacts? What about parameters for OutputUri?